### PR TITLE
localize `content/bn/releases/patch-releases.md` 

### PR DESCRIPTION
**Pull request by itsPronay**
Translated `patch-releases.md` for Bengali localization launch

file - `content/bn/releases/patch-releases.md`
mentioned in issue no #44993

fixes issue : #44993 
Related PR : #45009 
Tracking Issue: #31677

### CODE REVIEW

File: content/bn/releases/patch-releases.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+title: প্যাচ রিলিজ
+type: docs
+---
+
+কুবার্নিটিস প্যাচ রিলিজের সময়সূচি এবং দলের যোগাযোগ তথ্য।
```
Comment (asem-hamid) [inline content/bn/releases/patch-releases.md:6]: কুবারনেটিস

File: content/bn/releases/patch-releases.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+কুবার্নিটিস রিলিজ সাইকেলের সাধারণ তথ্যের জন্য, [রিলিজ প্রক্রিয়া বর্ণনা][release process description] দেখুন।
```
Comment (asem-hamid) [inline content/bn/releases/patch-releases.md:8]: কুবারনেটিস

**Comment by itsPronay** [comment]
ok, let me fix that up

File: content/bn/releases/patch-releases.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+## সংলগ্নতা
```
Comment (asem-hamid) [inline content/bn/releases/patch-releases.md:10]: ক্যাডেন্স (Cadence)

File: content/bn/releases/patch-releases.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+আমাদের সাধারণ প্যাচ রিলিজ সংলগ্নতা মাসিক। এটা
```
Comment (asem-hamid) [inline content/bn/releases/patch-releases.md:12]: ক্যাডেন্স

**Comment by asem-hamid** [comment]
@itsPronay can you join us at the [slack](https://kubernetes.slack.com/archives/CQ0TD298C)  it will help us to communicate .


Also, I haven't finished reviewing yet. Once I'm done, you can make any necessary fixes

**Comment by asem-hamid** [comment]
@itsPronay i guess you  have sync your branch with main ,  this PR is invalid now , close this and make a new PR with a new branch with your work .

**Pull request closed without merging**